sass/_themes.scss

```diff
@@ -218,7 +218,9 @@ $themes: (
         ChartHistoryPickerBodyActiveBg: $COLOR_DARK_BLUE_5,
         ChartHistoryPickerBodyActiveColor: $COLOR_WHITE,
         ChartHistoryPickerBodyActiveBorder: $COLOR_DARK_BLUE_5,
-        PaginationLoaderStripeColor: $COLOR_LIGHT_BLACK_4_SOLID
+        PaginationLoaderStripeColor: $COLOR_LIGHT_BLACK_4_SOLID,
+        WidgetBg: $COLOR_LIGHT_BLACK_4_SOLID,
```

Comment:
A bit nit picky but I think `WidgetBackground` would be a bit more readable or at least `WidgetBG`

Reply:
yep, as other word goes this way, I just try to follow the rules